static/app/views/monitors/components/monitorForm.tsx

```diff
@@ -90,10 +88,18 @@ function transformData(_data: Record<string, any>, model: FormModel) {
   }, {});
 }
 
-class MonitorForm extends Component<Props> {
-  form = new FormModel({transformData});
+function MonitorForm({
+  monitor,
+  submitLabel,
+  apiEndpoint,
+  apiMethod,
+  onSubmitSuccess,
+}: Props) {
+  const form = useRef(new FormModel({transformData}));
+  const {projects} = useProjects();
+  const {selection} = usePageFilters();
```

Comment:
oh why does monitor form need this actually?

Reply:
oh I think I see what you're doing

Reply:
oh yea, this was just carried over from before where we try to select the project that you already had selected previously. It's kinda buggy so i'll fix it up as im improving the form according to the new designs